tests/test_lazyflow/test_operators/test_ioOperators/testOpTiffReader.py

```diff
@@ -73,30 +78,15 @@ def test_unknown_axes_tags(self):
         import tifffile
         from distutils import version
 
-        # TODO(Dominik) remove version checking once tifffile dependency is fixed
-        # ilastik tiffile version >= 2000.0.0
-        # latest tifffile version is 0.13.0 right now
-        tifffile_version_ilastik_ref = version.StrictVersion("2000.0.0")
-        tifffile_version_ref = version.StrictVersion("0.7.0")
-        tifffile_version = version.StrictVersion(tifffile.__version__)
-
-        testshapes = [((10, 20), "yx"), ((10, 20, 30), "zyx"), ((10, 20, 30, 3), "zyxc"), ((5, 10, 20, 30, 3), "tzyxc")]
-
         with tempdir() as d:
-            for test_shape, test_axes in testshapes:
-                data = numpy.random.randint(0, 256, test_shape).astype(numpy.uint8)
-                tiff_path = "{}/myfile_{}.tiff".format(d, test_axes)
-                # TODO(Dominik) remove version checking once dependencies for
-                # skimage are >= 0.13.0 for all flavours of ilastik
-                if (tifffile_version > tifffile_version_ilastik_ref) or (tifffile_version < tifffile_version_ref):
-                    tifffile.imsave(tiff_path, data)
-                else:
-                    tifffile.imsave(tiff_path, data, metadata={"axes": "QQQ"})
-                op = OpTiffReader(graph=Graph())
-                op.Filepath.setValue(tiff_path)
-                assert op.Output.ready()
-                assert_array_equal(data, op.Output[:].wait())
-                assert op.Output.meta.axistags == vigra.defaultAxistags(test_axes)
+            data = numpy.random.randint(0, 256, test_shape).astype(numpy.uint8)
+            tiff_path = "{}/myfile_{}.tiff".format(d, axisorder)
```

Comment:
out of interest, on what operating system will it not work?

Reply:
This will probably work on Windows, but I think it's better to avoid explicit `/`s unless a library specifically says that it converts paths automatically. I doubt `tifffile` does that.